test/actions/pulp3/orchestration/remove_orphans_test.rb

```diff
@@ -39,6 +39,8 @@ def setup
 
       sync_and_reload_repo(@repo, @primary)
 
+      Setting[:completed_pulp_task_protection_days] = 0
+      DateTime.expects(:now).returns(DateTime.new(3000,1,1))
```

Comment:
```suggestion
      DateTime.expects(:now).returns(DateTime.new(3000, 1, 1))
```
Rubocop unhappy..Code looks good..Should be good to go if tests are happy..

Reply:
How rude of rubocop, I'll fix it.